### ISSUE TITLE
Unable to access setup on Wordpress site

### ISSUE THREAD

**Issue by MarieAnnick** [geoffhumphrey/brewcompetitiononlineentry#1204]
Version: 2.2.0

Installation URL: 138.197.150.223/abc/

Is your installation hosted on brewcompetition.com or brewcomp.com? no

Description of Issue: Previously ran the software on a straight debian setup, but they want the software hosted on a wordpress platform for a particular plugin. Attempting to access index.php or setup.php returns a 404 error. So far, I've attempted to alter the .htaccess file with and without the /abc subfolder and adding a base folder in config.php as well as deleting the .htaccess file since WP has its own about three directories back. I'm sure this is something as simple as putting the folder in the right place, but Wordpress documentation seems to pretend that people don't want to run non-wordpress stuff on their domains and is unhelpful in this regard. 

For enhancements, please prepend "Enhancement - " to the title. For issues/bugs, please prepend the BCOE&M version number the title.

**Comment by ercousin** [comment]
I use subdomain for BCOEM installs our wordpress site. Does a better job of avoiding collision with WP:

Eg.
https://brewslam.gtabrews.ca/

**Comment by MarieAnnick** [comment]
I figured it was going to be something like that. Do I have to mess around
with a new SQL database or can I use the existing one?

**Comment by ercousin** [comment]
I usually do a fresh install of the latest stable BCOEM version and create a new SQL database per comp, but at the same subdomain address. That way I have archives of past comps.

**Comment by bigirishape** [comment]
@ercousin What process do you follow to setup BCOEM alongside Wordpress?  I'm having some issues, and I hate to tag into an old thread, but getting a bit desperate and crowdsourced support for BCOEM doesn't seem to exist...

**Comment by MarieAnnick** [comment]
We set up a separate website specifically for the BCOEM and the database.
WordPress doesn't play nice with it.

**Comment by bigirishape** [comment]
Seems I fixed it.  I was struggling with the vhost setup for the subdomain, but got it sorted.  Also had to reconfigure the bcoem config to not use a subdirectory, and fix permissions, yadda yadda...

Pain in the rear :(

**Comment by ercousin** [comment]
Yep! I use a subdomain and it works great with wordpress on my main domain.

I can also host many BCOEM installs at once by giving a subdomain for each.
